webxrlayers-1.bs

```diff
@@ -446,13 +447,14 @@ The {{XRTextureType}} enum defines what type of texture is allocated.
 <pre class="idl">
 enum XRTextureType {
   "texture",
-  "texture-array"
+  "texture-array",
+  "cubemap-texture"
```

Comment:
I am not sure we need publicly expose "cubemap-texture". The cube layer may use only that, but we do not specify it the creation parameters.

Reply:
You're right. I mistakenly thought it was an attribute. 
I will fix.